.gitignore

```diff
@@ -23,3 +23,5 @@ includes/constants.php
 
 # Ignore JavaScript / Node.js -related files & directories
 node_modules/
+.ftpconfig
```

Comment:
What are these two ignores? What tool adds them to your instance and why are they needed (but unwanted in the repo)?

Reply:
Sorry, my .gitignore should not have been committed. You can ignore this change of .gitgnore.

Reply:
Ok, in this case, please remove these changes from your PR.